oteapi_dlite/strategies/oceanlab_influx_parser.py

```diff
@@ -0,0 +1,253 @@
+        "resource/url",
+        description=ResourceConfig.model_fields["resourceType"].description,
+    )
+    downloadUrl: Optional[HostlessAnyUrl] = Field(
+        None,
+        description=ResourceConfig.model_fields["downloadUrl"].description,
+    )
+    mediaType: Optional[str] = Field(
+        None,
+        description=ResourceConfig.model_fields["mediaType"].description,
+    )
+    storage_path: Annotated[
+        Optional[str],
+        Field(
+            description="Path to metadata storage",
+        ),
+    ] = None
+    collection_id: Annotated[
+        Optional[str], Field(description="A reference to a DLite collection.")
+    ] = None
+
+    url: Annotated[Optional[str], Field(description="url to the db")] = None
+
+    USER: Annotated[Optional[str], Field(description="user to the db")] = None
+
+    PASSWORD: Annotated[
+        Optional[str], Field(description="user pwd to the db")
+    ] = None
+
+    DATABASE: Annotated[Optional[str], Field(description="database name")] = (
+        None
+    )
+
+    RETPOLICY: Annotated[
+        Optional[str], Field(description="retpolicy for db")
+    ] = None
+
+
+class InfluxParseStrategyConfig(ParserConfig):
+    """DLite excel parse strategy  config."""
+
+    configuration: Annotated[
+        InfluxParseParseConfig,
+        Field(description="DLite json parse strategy-specific configuration."),
+    ]
+
+
+class InfluxParseSessionUpdate(DLiteSessionUpdate):
+    """Class for returning values from DLite json parser."""
+
+    inst_uuid: Annotated[
+        str,
+        Field(
+            description="UUID of new instance.",
+        ),
+    ]
+    label: Annotated[
+        str,
+        Field(
+            description="Label of the new instance in the collection.",
+        ),
+    ]
+
+
+@dataclass
+class InfluxParseStrategy:
+    """Parse strategy for Influx DB.
+
+    **Registers strategies**:
+
+    - `("parserType",
+        "json/vnd.dlite-json")`
+
+    """
+
+    parse_config: InfluxParseStrategyConfig
+
+    def initialize(self) -> DLiteSessionUpdate:
+        """Initialize."""
+        collection_id = (
+            self.parse_config.configuration.collection_id
+            or get_collection().uuid
+        )
+        return DLiteSessionUpdate(collection_id=collection_id)
+
+    def get(self) -> InfluxParseSessionUpdate:
+        """Execute the strategy.
+
+        This method will be called through the strategy-specific endpoint
+        of the OTE-API Services.
+
+        Returns:
+            DLite instance.
+
+        """
+        config = self.parse_config.configuration
+        try:
+            # Update dlite storage paths if provided
+            if config.storage_path:
+                for storage_path in config.storage_path.split("|"):
+                    dlite.storage_path.append(storage_path)
+        except Exception as e:
+            print(f"Error during update of DLite storage path: {e}")
```

Comment:
Should really be a log message instead of a print-statement :)
The same is true for the other print-statement(s) here.